kart/dataset_mixins.py

```diff
@@ -160,6 +166,66 @@ def get_decoder(dataset, method_name):
     _INSERT_UPDATE = _INSERT_TYPES + _UPDATE_TYPES
     _UPDATE_DELETE = _UPDATE_TYPES + _DELETE_TYPES
 
+    def get_raw_deltas_for_keys(
+        self, other, key_encoder_method, key_filter, reverse=False
```

Comment:
type hints on args wouldn't go amiss, the names don't make them very obvious

(we should probably be moving towards type hints everywhere)

Reply:
Done - added type hints to this file